x/tokenfactory/keeper/keeper_test.go

```diff
@@ -28,13 +29,11 @@ func (suite *KeeperTestSuite) SetupTest() {
 	suite.Setup()
 
 	// Fund every TestAcc with 100 denom creation fees.
-	fundAccsAmount := sdk.NewCoins(sdk.NewCoin(types.DefaultParams().DenomCreationFee[0].Denom, types.DefaultParams().DenomCreationFee[0].Amount.MulRaw(100)))
+	fundAccsAmount := sdk.NewCoins(sdk.NewCoin(types.DefaultParams().DenomCreationFee[0].Denom, types.DefaultParams().DenomCreationFee[0].Amount.MulRaw(100)), sdk.NewCoin("uion", sdk.NewInt(100000000)))
```

Comment:
Is it possible to avoid hardcoding "uion" since we are planning to use this denom across the test suite?

I find that in some modules there are various hardcoded denoms in the suite setup. As a result, it is difficult to search and determine which denom is used in which test.

If we move it to a shared variable, the search experience would be much easier. Also, other tests would be more likely to reuse it.

Reply:
Done, added a SecondaryDenom and SecondaryAmount in apptesting that we can start using in all other tests. Lmk if this addresses your concern!